Packs/CTFCloud/Dashboards/dashboard-CTF_Cloud_Dashboard.json

```diff
@@ -0,0 +1,451 @@
+            "x": 0,
+            "y": 1,
+            "i": "b72a3c70-3213-11ee-953d-71791f2973d4",
+            "w": 2,
+            "h": 1,
+            "widget": {
+                "id": "4fc2d3d6-a22c-4cf2-89f1-090147f42355",
+                "version": 1,
+                "cacheVersn": 0,
+                "modified": "2023-07-23T12:14:21.075546199Z",
+                "packID": "",
+                "packName": "",
+                "itemVersion": "",
+                "fromServerVersion": "",
+                "toServerVersion": "",
+                "propagationLabels": [
+                    "all"
+                ],
+                "definitionId": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "commitMessage": "",
+                "shouldCommit": false,
+                "Cache": null,
+                "name": "Number of Finished - Advanced",
+                "prevName": "Finished",
+                "dataType": "incidents",
+                "widgetType": "number",
+                "query": "advancedctfsla.runStatus:ended",
+                "isPredefined": false,
+                "dateRange": {
+                    "fromDate": "0001-01-01T00:00:00Z",
+                    "toDate": "0001-01-01T00:00:00Z",
+                    "period": {
+                        "by": "",
+                        "byTo": "",
+                        "byFrom": "days",
+                        "toValue": null,
+                        "fromValue": 7,
+                        "field": ""
+                    },
+                    "fromDateLicense": "0001-01-01T00:00:00Z"
+                },
+                "params": {
+                    "valuesFormat": "abbreviated"
+                },
+                "category": ""
+            },
+            "reflectDimensions": true
+        },
+        {
+            "id": "18defbc0-3216-11ee-953d-71791f2973d4",
+            "forceRange": false,
+            "x": 8,
+            "y": 0,
+            "i": "18defbc0-3216-11ee-953d-71791f2973d4",
+            "w": 4,
+            "h": 2,
+            "widget": {
+                "id": "79900405-6299-45ae-83e6-3279bd0fed11",
+                "version": 1,
+                "cacheVersn": 0,
+                "modified": "2023-07-23T12:16:55.375317441Z",
+                "packID": "",
+                "packName": "",
+                "itemVersion": "",
+                "fromServerVersion": "",
+                "toServerVersion": "",
+                "propagationLabels": [
+                    "all"
+                ],
+                "definitionId": "",
+                "vcShouldIgnore": false,
+                "vcShouldKeepItemLegacyProdMachine": false,
+                "commitMessage": "",
+                "shouldCommit": false,
+                "Cache": null,
+                "name": "Fastest Top 10 users finished Advanced CTF",
+                "prevName": "Top 10 users finished",
```

Comment:
Capitalized?
Top Ten Users to Finish

Reply:
No need, that is the previous name -> name is the right field to look at it.